flow/peerdbenv/config.go

```diff
@@ -156,3 +157,11 @@ func PeerDBGetIncidentIoUrl() string {
 func PeerDBGetIncidentIoToken() string {
 	return GetEnvString("PEERDB_INCIDENTIO_TOKEN", "")
 }
+
+func PeerDBRAPIRequestLoggingEnabled() bool {
+	requestLoggingEnabled, err := strconv.ParseBool(GetEnvString("PEERDB_API_REQUEST_LOGGING_ENABLED", "false"))
+	if err != nil {
```

Comment:
should log error

Reply:
I think then I should do it for oauth as well, will do it